src/functions/getATFsLambda.js

```diff
@@ -1,14 +1,21 @@
-const getATFList = require('../services/getATFListService')
+const ATFService = require('../services/ATFService')
+const ATFDAO = require('../models/ATFDAOmock')
+const path = require('path')
 
 const getATFs = async () => {
-  return getATFList()
+  const DAO = new ATFDAO(path.resolve(__dirname, '../../src/mocks/mock-atf.json'))
```

Comment:
this will be removed as part of next story which is about converting these mocks into facade services and pulling data from database